client/src/store/new-request-module.ts

```diff
@@ -2646,7 +2646,10 @@ export class NewRequestModule extends VuexModule {
       if (!this.noCorpNum) {
         const profile = await this.fetchMRASProfile()
         if (profile) {
-          name = sanitizeName(profile?.LegalEntity?.names?.legalName)
+          const hasMultipleNames = profile?.LegalEntity?.names && profile?.LegalEntity?.names.constructor === Array
+          name = hasMultipleNames
+            ? sanitizeName(profile?.LegalEntity?.names[0]?.legalName)
+            : sanitizeName(profile?.LegalEntity?.names?.legalName)
```

Comment:
Please call this out as a required QA test in https://github.com/bcgov/entity/issues/6028.

Reply:
For sure, will do!

Reply:
Done